.github/workflows/CI.yml

```diff
@@ -13,24 +13,27 @@ jobs:
   build-and-test:
 
     runs-on: windows-latest
-
+    strategy:
+      fail-fast: false
+      matrix:
+        browser: [chrome,firefox]
     steps:
     - uses: actions/checkout@v2
     - name: Set up JDK 11
       uses: actions/setup-java@v2
       with:
-        java-version: '11'
+        java-version: '8'
         distribution: 'adopt'
         cache: maven
     - name: Build with Maven
-      run: mvn -B test --file pom.xml
+      run: mvn -B test --file pom.xml -DargLine="-DexeBrowser=${{matrix.browser}}" cluecumber-report:reporting
     - uses: actions/download-artifact@v2
       with:
-        path: './target/HtmlReports.html'
+        path: './target/generatedreport'
     - name: Display structure of downloaded files
       run: ls -R
       working-directory: './target'
     - uses: actions/upload-artifact@v2
       with:
-        name: my-reportTest
-        path: './target/HtmlReports.html'
+        name: 'my-reportTest${{matrix.browser}}'
```

Comment:
I don't understand these changes

Reply:
I change the report, it is generated in a folder. Variable is sent to differentiate the reports